src/components/cart/components/cartCheckout/cartCheckout.constants.js

```diff
@@ -0,0 +1 @@
+export const LOCAL_STORAGE_KEY = "CartItems";
```

Comment:
LOCAL_STORAGE_KEY is not a good name for this because you can have many different key and value pairs to save in the localStorage